extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -191,6 +191,16 @@
               "type": "integer",
               "description":
                 "The delivery template's ID used to initialize this delivery.\n\nThe type the template used in Adobe Campaign can be identified using the `messageClass`.\n\n* `messageClass` = `oneTime`: the template is standard delivery template.\n* `messageClass` = `continuous`: the template is a recurring delivery.\n* `messageClass` = `transactional`: the template is a transactionnal message template.\n"
+            },
+            "xdm:deliveryLabel": {
+              "title": "Delivery Label",
+              "type": "string",
+              "description" : "Human friendly name of the campaign activity which is originating this message."
+            },
+            "xdm:deliveryName": {
```

Comment:
I still think the term _delivery_ is not needed here and in the  `xdm:deliveryLabel` since they are part of the _delivery_ schema/object.

Otherwise you could also make the case to call the ID field `xdm:deliveryId`.

Reply:
change from "name", "label" to deliveryName and "deliveryLabel" is requested in one of the review comments. 
https://github.com/adobe/xdm/pull/464#discussion_r207927635